examples/tools/auto_cube_disconnect.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Set whether Cozmo should auto-disconnect from cubes after each SDK program.
+
+This script can be used to turn auto cube disconnection on or off (using args)
+Auto disconnection can be used to help automatically conserve cube battery life
```

Comment:
Above you have a hyphen in 'auto-disconnect'. Should also have one in auto-disconnection?